### PR TITLE
Authentication beach point 

### PR DESCRIPTION
**Pull request by aristi1215**

### CODE REVIEW

File: src/adminDashboard/pages/DashboardAdmin.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    navigate('/')
+  }
+  return (
+      <div>
+        <Toaster />
+        <h1>
+            Dashboard
+        </h1>
+        <button className='bg-black text-white' onClick={() => handleLogout()}>
+                Sali de aca bobo
```
Comment (DoseiBrayin) [inline src/adminDashboard/pages/DashboardAdmin.jsx:22]: ????

**Review by DoseiBrayin — COMMENTED** [review]
Ese boton que?

File: src/auth/SignUp/pages/SignUp.jsx [hunk 2]
```diff
@@ -2,24 +2,63 @@ import { Footer } from '../../../LandingPage/components/Footer'
...
-  const validationRules = getValidationRules(getValues)
+  const handleRegister = async (formData) => {
+    if (formData.name && formData.password && formData.confirm_pwd && formData.email) {
+      await registerUser({ id: v4(), cedula: formData.ID, name: formData.name, password: formData.password, email: formData.email, phone_number: '3013782590', card_id: '', is_employee: false, fk_rol: 'USR' })
+    }
+  }
+
+  useEffect(() => {
+    if (isSuccess) {
+      console.log('hola exitoso')
```
Comment (DoseiBrayin) [inline src/auth/SignUp/pages/SignUp.jsx:41]: Este console.log????

File: src/auth/Login/pages/Login.jsx [hunk 3]
```diff
@@ -2,25 +2,56 @@ import { Footer } from '../../../LandingPage/components/Footer'
...
+  useEffect(() => {
+    if (isSuccess) {
+      toast.success('User loged in successfully')
+      dispatch(
+        setUser({ token: data.data })
+      )
+      navigate('/AdminDashboard')
+    }
+    if (isError) {
+      console.log(error.data)
```
Comment (DoseiBrayin) [inline src/auth/Login/pages/Login.jsx:41]: console log??

**Review by DoseiBrayin — CHANGES_REQUESTED** [review]

**Comment by aristi1215** [comment]
Ya no hay consoles 🙏🙈

**Pull request merged**